doc/releases/changelog-0.9.0.md

```diff
@@ -50,9 +30,12 @@
   >>> circuit()
   (Array([[0], [1], [0], [1], [1]], dtype=int64),
   Array([[0], [1], [1], [0], [1]], dtype=int64),
-  Array([[1], [0], [1], [1], [0], [1],[0]], dtype=int64))
+  Array([[1], [0], [1], [1], [0], [1], [0]], dtype=int64))
   ```
 
+  Note that other measurement types, such as `expval` and `probs`, currently
+  do not support shot-vectors.
+
 * A new function `catalyst.passes.pipeline` allows the quantum-circuit-transformation pass pipeline
```

Comment:
@paul0403 I noticed that this function deviates from the requirements; in the requirements it is `catalyst.pipeline`. Would we be able to move it top level?

Reply:
I will add an alias at top level.

Reply:
thank you!